pytorch_lightning/utilities/fetching.py

```diff
@@ -228,44 +228,33 @@ def wait(self) -> None:
         """Hook to override to indicate the `DataFetcher` to wait for an event."""
 
     def prefetching(self) -> None:
+        iterator = self.dataloader_iter
         for _ in range(self.prefetch_batches):
             try:
-                self._fetch_next_batch()
+                self._fetch_next_batch(iterator)
```

Comment:
```suggestion
                self._fetch_next_batch(self.dataloader_iter)
```
No need for a separate variable :)